0.18-rc: "‘for’ loop initial declarations are only allowed in C99 or C11 mode" compile time error

```
/home/constantine/Desktop/libmill/tests/signals.c:71:5: error: ‘for’ loop initial declarations are only allowed in C99 or C11 mode
     for(int i = 0; i < COUNT; ++i) {
     ^
/home/constantine/Desktop/libmill/tests/signals.c:71:5: note: use option -std=c99, -std=gnu99, -std=c11 or -std=gnu11 to compile your code
```

Makefile was generated from CMake.

--- a/tests/signals.c
+++ b/tests/signals.c
@@ -68,7 +68,8 @@ int main() {
     chan sendch = chmake(char, 0);
     chan recvch = chmake(char, 0);
 
-    for(int i = 0; i < COUNT; ++i) {
+    int i;
+    for(i = 0; i < COUNT; ++i) {
         go(sender(sendch));
         go(receiver(recvch));
         chs(sendch, char, SIGNAL);

